Peer review: Henry Ford

hgojf opened the pull request:
Site is at: [https://hgojf.github.io/queens-student-page/](https://hgojf.github.io/queens-student-page/)

hgojf committed:
Add a TODO

Helen-1015 committed:
Update index.html

Helen-1015 commented on index.html:
```diff
@@ -27,7 +28,7 @@ <h2>Courses</h2>
         <li>CISC 102 — Discrete structures.</li>
         <li>CISC 121 — Intro to computing science.</li>
         <li>MATH 110 — Linear algebra.</li>
-        <li>MATH 120 — Differentiel and integral calculus.</li>
```
The spelling of the differentiel actually is differential

Helen-1015 commented on index.html:
```diff
@@ -27,7 +28,7 @@ <h2>Courses</h2>
         <li>CISC 121 — Intro to computing science.</li>
         <li>MATH 110 — Linear algebra.</li>
-        <li>MATH 120 — Differentiel and integral calculus.</li>
+        <li>MATH 120 — Differentiel and integral calculus.</li> Differentiel -- Differential
```
This problem is same like the last one

Helen-1015 reviewed (commented):
Functionality: Very clear and every link can open quickly with no errors
Clarity: Wonderful.
Structure: Headings are used consistently (#, ##), making the page easy to follow.
Design: Clean and simple layout. The page loads quickly and has no obvious errors.
Deployment: GitHub Pages works properly; the site is live and accessible.
In About Me, add a bit more detail (e.g., which areas of computing/mathematics interest you most).
In Courses, fix the typo: Differentiel → Differential.

hgojf committed:
Fix spelling of differential

Fix from Hailin Zhang, thanks!

hgojf committed:
Remove vague sentence.

I'm not sure what areas of computing and mathematics I'm interested
in yet, so this doesn't really provide valuable information.
